scripts/android-sdk-update.sh

```diff
@@ -2,21 +2,33 @@
 
 # This script will update
 # Android tools, build tools and platform tools
-# Android SDKs from 4.2 (android-17) to 7.1 (android-25)
+# Android SDKs from 4.2 (android-17) to 8.1 (android-27)
 #
 # Notes: This script will work on both Linux and macOS hosts!
 
+set -e
+set -x
+
+PACKAGES=()
+PACKAGES+=("platform-tools")
+PACKAGES+=("tools")
+PACKAGES+=("build-tools;27.0.3")
+PACKAGES+=("extras;android;m2repository")
+PACKAGES+=("extras;google;m2repository")
+PACKAGES+=("platforms;android-7")
+PACKAGES+=("platforms;android-8")
+PACKAGES+=("platforms;android-9")
```

Comment:
`android-7`, `android-8` and `android-9` are not necessary

Reply:
Will remove those